src/userCLI/build.gradle

```diff
@@ -36,6 +48,16 @@ if (hasProperty('docker_name')) {
 distDocker {
     baseImage = "openjdk:8u121-jre"
     dependsOn = [build]
+    push = false
     tag = cli_docker_name
     tagVersion = cli_docker_version
 }
+
+test {
+    useJUnitPlatform()
+}
+
+jar{
```

Comment:
This should already be enabled in the root gradle config's "subprojects" block; you should be able to move the `configurations` and `test` blocks into there as well since they're shared by both subprojects